modules/configure-an-additional-clientCA.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+//
+// * security/certificates/api-server.adoc
+
+:_content-type: PROCEDURE
+[id="configure-an-additional-clientCA-for-the-OpenShift-API-server_{context}"]
+
+= Configure an additional clientCA for the OpenShift API server
+
+Optionally, you may choose to invalidate the installer-generated kubeconfig. You would do this when:
+
+* You don't trust who installed the cluster
```

Comment:
```suggestion
* You do not trust who installed the cluster
```

Reply:
Why would you not trust who installed the cluster?

Reply:
- Spelled out "do not."
- If you don't know the certifying authority, you might feel more secure generating a certificate from a known and trusted CA.